pages/linux/vnstati.md

```diff
@@ -0,0 +1,20 @@
+# vnstati
+
+> PNG image output support for vnStat.
+> More information: <https://manned.org/vnstati>.
+
+- Output a summary of the last 2: months, days, and all time:
+
+`vnstati --summary --iface {{network_interface}} --output {{output.png}}`
+
+- Output all time top 10 traffic days:
+
+`vnstati --top10 --iface {{network_interface}} --output {{output.png}}`
+
+- Output traffic statistics on a monthly basis for the last 12 months:
+
+`vnstati --months --iface {{network_interface}} --output {{output.png}}`
+
+- Output traffic statistics on a hourly basis for the last 24 hours:
```

Comment:
```suggestion
- Output traffic statistics on an hourly basis for the last 24 hours:
```

Should be "an" here, not "a"!

Reply:
You are right, but I got that title from the man page, so they made the mistake, not me. 😅